src/options/modules/CustomOptionTriggers.js

```diff
@@ -204,10 +219,12 @@ export function registerTrigger() {
     AutomaticSettings.Trigger.registerSave("qrColor", applyQrCodeColors);
     AutomaticSettings.Trigger.registerSave("qrBackgroundColor", applyQrCodeColors);
 
+    AutomaticSettings.Trigger.registerUpdate("qrQuietZone", updateQrQuietZoneStatus);
     AutomaticSettings.Trigger.registerUpdate("qrColor", applyQrCodeColors);
     AutomaticSettings.Trigger.registerUpdate("qrBackgroundColor", applyQrCodeColors);
 
     // handle loading of options correctly
     AutomaticSettings.Trigger.registerBeforeLoad(resetOnBeforeLoad);
+    AutomaticSettings.Trigger.registerAfterLoad(updateQrQuietZoneStatus);
```

Comment:
This is not needed. The exact line below this one does automatically register **all save triggers** for the load trigger, too, so you can safely remove this.

Reply:
okay, it was a little harder, but I've changed it now :smile: